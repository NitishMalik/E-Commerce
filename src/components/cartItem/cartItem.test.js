import React from "react";
import { shallow } from "enzyme";
import CartItem from "./index";

it("expect to render cart item", () => {
  // We have to create the props we are passing to the component
  const abcd = {
    item: {
      imageUrl: "http://google.com",
      price: 1,
      name: "Nitish",
      quantity: 1
    }
  };

  expect(shallow(<CartItem item={abcd} />)).toMatchSnapshot();
});
