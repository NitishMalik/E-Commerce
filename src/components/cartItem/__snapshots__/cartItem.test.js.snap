// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render cart item 1`] = `ShallowWrapper {}`;
